src/controllers/objecting.entity.name.controller.ts

```diff
@@ -16,6 +22,58 @@ const validators = [
   check(DIVULGE_INFO_FIELD).not().isEmpty().withMessage(ErrorMessages.SELECT_TO_DIVULGE),
 ];
 
+const getExistingPageData = async (session: Session, res: Response, next: NextFunction) => {
+  try {
+    const objection: Objection = await getObjection(session) as Objection;
+    const existingName: string = objection.created_by.fullName;
+    const existingShareIdneity: string = (objection.created_by.shareIdentity === true) ? "yes" : "no";
+    if (existingName && existingShareIdneity) {
+      return res.render(Templates.OBJECTING_ENTITY_NAME, {
+        fullNameValue: existingName,
+        isYesChecked: existingShareIdneity === "yes",
+        isNoChecked: existingShareIdneity === "no",
+        templateName: Templates.OBJECTING_ENTITY_NAME,
+      });
+    } else {
+      return next(new Error("Existing data not present"));
```

Comment:
we need to show the page with no data, as when a customer is completing the screens first time through, the data will not have been submitted. I haven't tested it but it looks like line 38 will show the error screen if there is no existing data.

Reply:
sorry, ignore above, i can see further down that it renders page normally

Reply:
It will this is when we expect there to be data and there is none - line 61 deals with completing the screens first time through. However we could just redner the page here as well and put an error in the logs - any thoughts?